guillotina/_settings.py

```diff
@@ -97,5 +97,12 @@
     "valid_id_characters": string.digits + string.ascii_lowercase + ".-_@$^()+ =",
     "load_catalog": True,
     "catalog_max_results": 50,
+    "managers_roles": {
```

Comment:
can we rename `managers_roles` -> `manager_roles`?

Reply:
Good idea!

Reply:
the group is called managers

Reply:
But they are `roles` a "Manager" will have. Dunno, just sounds funny to me. You can leave it.